lib/wrappers/openssl.nim

```diff
@@ -221,6 +232,18 @@ else:
       if not dl.isNil:
         result = symAddr(dl, name)
 
+  proc loadedOpenSSLBefore1_1*(): bool =
```

Comment:
I would prefer a ``getOpenSSLVersion`` procedure that returns a tuple instead. I think OpenSSL exposes that somewhere.

Reply:
I'm being told the version is not exposed in windows. If you can find a reliable way to get it I'd be happy to use that instead of this guesswork.

Reply:
Can't you simply wrap this? https://www.openssl.org/docs/manmaster/man3/OPENSSL_VERSION_NUMBER.html

Reply:
@dom96 OpenSSL_version_num is exported by libcrypto.so.1.1 but not libcrypto.so.1.0.2 and earlier

Reply:
So create a ``getVersion`` procedure that returns (1,0,2) when ``OpenSSL_version_num`` is unavailable. Explain this in the docs.

Reply:
OpenSSL_version_number + description added.

Reply:
Why is this still here? You should just use ``OpenSSL_version_number``